deny.toml

```diff
@@ -0,0 +1,25 @@
+[graph]
+all-features = true
+
+[licenses]
+allow = [
+    "Apache-2.0",
+    "Apache-2.0 WITH LLVM-exception",
+    "BSL-1.0",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "CC0-1.0",
+    "ISC",
+    "GPL-3.0",
+    "MIT",
+    "OpenSSL",
```

Comment:
It looks better to me to to remove OpenSSL from here ([resource](https://www.gnu.org/licenses/license-list.html#OpenSSL))

And add these lines at the bottom:
```rs
[[licenses.exceptions]]
allow = ["OpenSSL"]
name = "ring"
```

Reply:
Yup, much better, thanks!